sdk/storage/test-resources-post.ps1

```diff
@@ -164,16 +165,31 @@ $content =
       <VaultEndpoint>$KeyVaultUri</VaultEndpoint>
       <ActiveDirectoryApplicationId>$TestApplicationId</ActiveDirectoryApplicationId>
       <ActiveDirectoryApplicationSecret>$TestApplicationSecret</ActiveDirectoryApplicationSecret>
-      <ActiveDirectoryTenantId>$TenantId</ActiveDirectoryTenantId>
+      <ActiveDirectoryTenantId>$StorageTenantId</ActiveDirectoryTenantId>
       <ActiveDirectoryAuthEndpoint>https://login.microsoftonline.com/</ActiveDirectoryAuthEndpoint>
     </KeyVaultConfiguration>
   </KeyVaultConfigurations>
 </TestConfigurations>"
 
+$storageTestConfigurationTemplateName = 'TestConfigurationsTemplate.xml'
+$storageTestConfigurationName = 'TestConfigurations.xml'
+
+if(-not (Test-Path $env:BUILD_ARTIFACTSTAGINGDIRECTORY -ErrorAction Ignore) ) { 
+  Write-Verbose "Checking for '$storageTestConfigurationTemplateName' files under '$PSScriptRoot'"
+
+  $foundFile = Get-ChildItem -Path $PSScriptRoot -Filter $storageTestConfigurationTemplateName -Recurse | Select-Object -First 1 
+  $storageTemplateDirName = $foundFile.Directory.FullName
+  Write-Verbose "Found template dir '$storageTemplateDirName'"
+    
+} else {
+  $storageTemplateDirName = $env:BUILD_ARTIFACTSTAGINGDIRECTORY
+  Write-Verbose "Found environment variable BUILD_ARTIFACTSTAGINGDIRECTORY '$storageTemplateDirName'"
+}
+

```

Comment:
@kasobol-msft This should fix local running of New-TestResources.ps1 (and I don't think it should adversely affect nightly CI)

Reply:
Nice. `/azp run net - storage - tests` will tell us whether nightly run is affected or not.